client/my-sites/activity/activity-log-confirm-dialog/index.jsx

```diff
@@ -47,46 +46,44 @@ const ActivityLogConfirmDialog = ( {
 
 			<div className="activity-log-confirm-dialog__highlight">{ children }</div>
 
-			{ config.isEnabled( 'rewind/partial-restores' ) && (
-				<Card className="activity-log-confirm-dialog__partial-restore-settings">
-					<p>
-						<strong>
-							{ notice
-								? translate( 'Partial Restore Settings (A8C Only)' )
-								: translate( 'Partial Download Settings (A8C Only)' ) }
-						</strong>
-					</p>
-					<p>
+			<Card className="activity-log-confirm-dialog__partial-restore-settings">
+				<p>
+					<strong>
 						{ notice
-							? translate( 'Include the following things in this restore:' )
-							: translate( 'Include the following things in this download:' ) }
-					</p>
-					<FormLabel>
-						<FormCheckbox name="themes" onChange={ onSettingsChange } defaultChecked />
-						{ translate( 'WordPress Themes' ) }
-					</FormLabel>
-					<FormLabel>
-						<FormCheckbox name="plugins" onChange={ onSettingsChange } defaultChecked />
-						{ translate( 'WordPress Plugins' ) }
-					</FormLabel>
-					<FormLabel>
-						<FormCheckbox name="uploads" onChange={ onSettingsChange } defaultChecked />
-						{ translate( 'Media Uploads' ) }
-					</FormLabel>
-					<FormLabel>
-						<FormCheckbox name="roots" onChange={ onSettingsChange } defaultChecked />
-						{ translate( 'WordPress Root (includes wp-config.php and any non-WordPress files)' ) }
-					</FormLabel>
-					<FormLabel>
-						<FormCheckbox name="contents" onChange={ onSettingsChange } defaultChecked />
-						{ translate( 'WP-Content Directory (excluding themes, plugins, and uploads)' ) }
-					</FormLabel>
-					<FormLabel>
-						<FormCheckbox name="sqls" onChange={ onSettingsChange } defaultChecked />
-						{ translate( 'Site Database (SQL)' ) }
-					</FormLabel>
-				</Card>
-			) }
+							? translate( 'Partial Restore Settings' )
+							: translate( 'Partial Download Settings' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Partial Download Settings (A8C Only)' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).